clang/include/clang/Basic/BuiltinsRISCV.td

```diff
@@ -147,6 +147,13 @@ def ntl_load : RISCVBuiltin<"void(...)">;
 def ntl_store : RISCVBuiltin<"void(...)">;
 } // Features = "zihintntl", Attributes = [CustomTypeChecking]
 
+//===----------------------------------------------------------------------===//
+// Zihintpause extension.
+//===----------------------------------------------------------------------===//
+let Features = "zihintpause" in {
```

Comment:
This should have `Attributes = [NoThrow]`. Zihintntl should too, but that's for a different patch.

Reply:
Thanks. I will do the same for zihintntl in another PR